### PR TITLE
Add total coverage metric in summary

### PR DESCRIPTION
**Pull request by ciecierski**
This pr adds **Total FAILURE coverage** metric to the summary table.

The purpose of the metric is to show how many failing builds are not reported as jira/bz/other, in other words not tracked.

### CODE REVIEW

File: jeeves/blockers.py [hunk 1]
```diff
@@ -152,9 +152,7 @@ def get_other_blockers(blockers, job_name):
 		returns list of 'other' blockers
 	'''
 	other = []
-	other_blockers = blockers[job_name].get('other')
-	if other_blockers is None:
-		return other
+	other_blockers = blockers[job_name]['other']
```
Comment (nathan-weinberg) [inline jeeves/blockers.py:155]: What is the motivation behind this change?

File: jeeves/report.py [hunk 2]
```diff
@@ -101,17 +104,21 @@ def run_report(config, blockers, preamble_file, server, header, test_email, no_e
...
 				# get any "other" artifact for job
 				try:
 					other = get_other_blockers(blockers, job_name)
+					job_covered = True
 				except Exception as e:
 					print("Error fetching other blockers for job {}: {}".format(job_name, e))
 					other = []
 
+				if job_covered and jenkins_api_info['lcb_result'] == "FAILURE":
+					num_covered += 1
```
Comment (nathan-weinberg) [inline jeeves/report.py:121]: Why only FAILURE jobs?

File: jeeves/report.py [hunk 3]
```diff
@@ -163,6 +170,7 @@ def run_report(config, blockers, preamble_file, server, header, test_email, no_e
 	summary['total_success'] = "Total SUCCESS:  {}/{} = {}%".format(num_success, num_jobs, percent(num_success, num_jobs))
 	summary['total_unstable'] = "Total UNSTABLE: {}/{} = {}%".format(num_unstable, num_jobs, percent(num_unstable, num_jobs))
 	summary['total_failure'] = "Total FAILURE:  {}/{} = {}%".format(num_failure, num_jobs, percent(num_failure, num_jobs))
+	summary['total_coverage'] = "Total FAILURE coverage:  {}/{} = {}%".format(num_covered, num_failure, percent(num_covered, num_failure))
```
Comment (nathan-weinberg) [inline jeeves/report.py:173]: Again why only FAILURE jobs? Wouldn't it be better to show overall coverage of non-passing jobs?

File: templates/report_template.html [hunk 4]
```diff
@@ -36,6 +36,7 @@
 							{% endif %}
 							<li>{{summary.total_bugs}}</li>
 							<li>{{summary.total_tickets}}</li>
+                            <li>{{summary.total_coverage}}</li>
```
Comment (nathan-weinberg) [inline templates/report_template.html:39]: Please indent to match the rest of the summary list items

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

File: jeeves/blockers.py (hunk 1 above)
Comment (nathan-weinberg) [reply jeeves/blockers.py:155]: Makes sense; the behavior appears to be the same either way so this is fine with me

File: jeeves/report.py (hunk 3 above)
Comment (itzikb) [reply jeeves/report.py:173]: +1

Comment (ciecierski) [reply jeeves/report.py:173]: I was thinking about removing failure constraint and making it more generic. How about total coverage(including aborted,failures and unstable)? Does it works for you?

Comment (nathan-weinberg) [reply jeeves/report.py:173]: Agreed; I would say let's track the metric for all jobs that don't have "SUCCESS" status, i.e. "UNSTABLE", "FAILURE", "ABORTED", "NO_KNOWN_BUILDS"

File: jeeves/report.py [hunk 5]
```diff
@@ -163,6 +170,7 @@ def run_report(config, blockers, preamble_file, template_file, no_email, test_em
 	summary['total_success'] = "Total SUCCESS:  {}/{} = {}%".format(num_success, num_jobs, percent(num_success, num_jobs))
 	summary['total_unstable'] = "Total UNSTABLE: {}/{} = {}%".format(num_unstable, num_jobs, percent(num_unstable, num_jobs))
 	summary['total_failure'] = "Total FAILURE:  {}/{} = {}%".format(num_failure, num_jobs, percent(num_failure, num_jobs))
+	summary['total_coverage'] = "Total bz/jira/other coverage:  {}/{} = {}%".format(num_covered, num_jobs - num_success, percent(num_covered, num_jobs - num_success))
```
Comment (nathan-weinberg) [inline jeeves/report.py:173]:
```suggestion
	summary['total_coverage'] = "Total Blocker Coverage:  {}/{} = {}%".format(num_covered, num_jobs - num_success, percent(num_covered, num_jobs - num_success))
```

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
One change request but otherwise LGTM

**Review by nathan-weinberg — APPROVED** [review]
